documents/openapi/openapi.yaml

```diff
@@ -434,13 +438,16 @@ paths:
                   minimum: 1
                 sms_sender:
                   type: string
+                  example: '+19876543210'
```

Comment:
this has a mexico area code. just to make sure that we aren't using any real numbers, can you make this a 555 (fictional) number? ex; `+12025551212`

Reply:
sure, it's just the numbers 0-9 in reverse order, but I'll update it. If it's a Mexico area code that would mean it has a different country code right?